charts/posthog/templates/clickhouse_instance.yaml

```diff
@@ -21,9 +21,13 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: gp2-resizable
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"   
```

Comment:
there can/should only be one default class per cluster. A normal user of this chart would not expect that the chart adds a default storage class, because setting up storage classes is what a cluster administrator does and not something an application administator would do.

Also creating non namespaced resources which are hidden in the helm chart are not best practice, because this can result in conflicts with existing resources in the cluster.

Reply:
Thanks. Note that this PR wasn't intended to be landed just to be clear. It was only used for a custom upgrade.

Reply:
+1 to what @Legion2 wrote. See also https://github.com/PostHog/charts-clickhouse/issues/174 for more info

Reply:
Can we close this without merging @tiina303 ?